config/tsconfig.build.json

```diff
@@ -51,7 +51,7 @@
     },
     "plugins": [{"name": "typescript-styled-plugin"}]
   },
-  "include": ["../static", "../tests/js", "../tests/fixtures/js-stubs"],
+  "include": ["../static", "../tests/fixtures/js-stubs"],
```

Comment:
Do we still run these for CI? cc @scttcper

Reply:
Ah, you know what. This is defined in `js-build-and-lint.yml`:
```
      - name: tsc
        if: steps.dependencies.outcome == 'success' && steps.changes.outputs.frontend == 'true'
        run: |
          set -o pipefail
          yarn tsc -p config/tsconfig.build.json --diagnostics | tee /tmp/typescript-monitor.log && yarn run ts-node .github/workflows/scripts/monitor-typescript.ts
```

We have a root `tsconfig.json` that still references the tests path, but it also sets `allowJs: true`, not sure if it would adversely affect our numbers if we target that config. Maybe the simpler thing I can do is move `react-select-event` into `dependencies`.

Reply:
> Maybe the simpler thing I can do is move react-select-event into dependencies

👍

Reply:
I just want to make sure we don't lose typechecking on tests!

Reply:
We should probably have a tsconfig for tests/ci separate from build